core/src/storage/device.py

```diff
@@ -3,6 +3,7 @@
 
 from storage import common
 from trezor import utils
+from trezor.enums import DisplayRotation
```

Comment:
please move this import to (a) TYPE_CHECKING block and (b) again locally inside the function

(bla bla generated bytecode + dependencies concerns)